src/ui/Base/Component.ts

```diff
@@ -226,6 +220,25 @@ export class ComponentEvents {
     };
   }
 
+  private shouldTreatElementAsJQuery(arg: any) {
+    if (Dom.useNativeJavaScriptEvents === true) {
+      return false;
+    }
+
+    if (JQueryUtils.getJQuery() && JQueryUtils.isInstanceOfJQuery(arg)) {
+      return true;
+    }
+
+    return false;
```

Comment:
Similar to the `shouldTreatEventAsJQuery` function, we could simplify to just return the conditional on line 228.

```
if (Dom.useNativeJavaScriptEvents === true) {
      return false;
}

return JQueryUtils.getJQuery() && JQueryUtils.isInstanceOfJQuery(arg)
```